engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/formatter/ConfluentAvroMessageReader.scala

```diff
@@ -24,7 +26,8 @@ import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentUtils
   * @param parseKey if key should be parsed
   * @param keySeparator key separator
   */
-private[confluent] class ConfluentAvroMessageReader(schemaRegistryClient: SchemaRegistryClient,
+private[confluent] class ConfluentAvroMessageReader(kafkaConfig: KafkaConfig,
```

Comment:
WDYT about passing  `useStringForKey` instead of full config here? It would be consistent with current approach for `parseKey` and `keySeparator`